frontend/src/app/team-management/new-user/new-user.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="d-flex w-100">
+  <div class="firstname equal-sized-col">
+    <div class="input-style">
+      @if (index === 0) {
+        <label [for]="'firstName-col_' + index">Vorname</label>
+      }
+      <input
+        required
+        minlength="1"
+        type="text"
+        class="value-field"
+        #firstInput
+        #firstName="ngModel"
+        [(ngModel)]="user.firstname"
+        [id]="'firstName-col_' + index"
+        [name]="'firstName-col_' + index"
+      />
+      @if (showError(firstName)) {
+        <mat-error>
+          @if (firstName.invalid) {
+            <span>Angabe benötigt</span>
+          }
+        </mat-error>
+      }
+    </div>
+  </div>
+  <div class="lastname equal-sized-col">
+    <div class="input-style equal-sized-col">
+      @if (index === 0) {
```

Comment:
Ich finde es ein bisschen speziell einen index mitzugeben. Ist die Label-click logik für das fokusieren des inputs notwendig? Sonst könnte man wahrscheinlich auch ohne index auskommen.
Und: Wenn es 2 Zeilen sind und ich auf das Label klicke, dann springt es in das input der ersten Zeile.

(Könnte man aber evtl. auch so lassen)

Reply:
Wenn du das nicht über Labels machst, müsstest du die Titel wohl im äusseren invide-user-dialog.component.html hinzufügen. Dann hätte man wiederum das Problem, dass das Styling vielleicht nicht stimmt und die Spaltenbreite irgendwie kopiert werden müsste. Oder was wäre dann die Idee?